sqa/eth/ingest/ingest.py

```diff
@@ -216,6 +218,14 @@ async def _fetch_blocks(self, from_block: int, to_block: int) -> list[Block]:
             for block in blocks:
                 for tx in block['transactions']:
                     tx['type'] = '0x0'
+        if self._is_bitfinity_mainnet:
+            for block in blocks:
+                for tx in block['transactions']:
+                    if tx.get('type') is None:
+                        if 'maxPriorityFeePerGas' in tx:
+                            tx['type'] = '0x2'
```

Comment:
i have an idea that this assuption about tx type shoul be a part of validation logic and not ingestion

Reply:
as i noticed many libraries that provide validation they just try to substitute missing values

Reply:
I don't think in this exact case it would be good to move to validation logic. But in general yes!